src/components/multiselect/MultiSelect.js

```diff
@@ -387,6 +387,10 @@ export class MultiSelect extends Component {
         return this.props.optionLabel ? ObjectUtils.resolveFieldData(option, this.props.optionLabel) : option.label;
     }
 
+    getOptionClassName(option) {
+        return this.props.optionLabel ? ObjectUtils.resolveFieldData(option, this.props.className) : option.className;
```

Comment:
`this.props.className`  should be replaced with a new prop like `this.props.optionClassName` and default it to `"className"` if not set.

I'm also not sure if it is wise to make `optionLabel` necessary to make the new prop `optionClassName` have an effect.

Reply:
You are right, optionsLabel requirement should be dropped, do you want to add another review?

Reply:
> do you want to add another review?

Sorry. I don't understand the question.

But as I said `this.props.className` in `MultiSelect.js` is inappropriate, because it is used for the actual class name for the main component. A new prop is required.

If we add a new prop then we also need to update `MultiSelect.d.ts`.